backend/controller/sql/schema/20240812011321_derive_encryption.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+
+CREATE TABLE encryption_keys (
+    id          BIGINT NOT NULL GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    key         bytea NOT NULL,
+    created_at  TIMESTAMPTZ NOT NULL DEFAULT (NOW() AT TIME ZONE 'utc')
+);
+
+ALTER TABLE events
+    DROP COLUMN payload,
+    ADD COLUMN payload bytea;
+
+ALTER TABLE async_calls
+    DROP COLUMN request,
+    ADD COLUMN request bytea;
```

Comment:
May as well do response here (and encrypt it) too? Just in case we end up using it.